test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -297,7 +297,7 @@ static inline void multi_access_publisher(bool intra_process)
     };
   std::vector<rclcpp::timer::TimerBase::SharedPtr> timers;
   // timers will fire simultaneously in each thread
-  for (uint32_t i = 0; i < std::min<unsigned int>(executor.get_number_of_threads(), 16); i++) {
+  for (uint32_t i = 0; i < std::min<size_t>(executor.get_number_of_threads(), 16); i++) {
```

Comment:
Ah, there was pre-existing non-conformant style.

Can the linters be made to check for that? It's not good a use of humans' time to check such things...

f060cd1